cla_backend/apps/cla_butler/tests/test_tasks.py

```diff
@@ -170,4 +171,55 @@ def test_delete_old_data_run_case_under_three_years_unsuccessful_delete(self):
         pks = get_pks(Case.objects.all())
         eods = EODDetails.objects.filter(case_id__in=pks).values_list("pk", flat=True)
 
-        self.data_deletion_check(pks, eods, 1, 1)
+        self.data_deletion_check(pks, eods, 1, 1, delete_option_three_years)
+
+    def test_delete_no_personal_details_success(self):
```

Comment:
this should be extended to allow for cases with notes, with provider_notes and without etc